packages/gatsby-source-contentful/src/fetch.js

```diff
@@ -148,7 +148,13 @@ function pagedGet(
       aggregatedResponse.items = aggregatedResponse.items.concat(response.items)
     }
     if (skip + pageLimit <= response.total) {
-      return pagedGet(client, method, skip + pageLimit, aggregatedResponse)
+      return pagedGet(
+        client,
+        method,
+        query,
+        skip + pageLimit,
```

Comment:
Function signature is:
```
function pagedGet(
  client,
  method,
  query = {},
  skip = 0,
  pageLimit = 1000,
  aggregatedResponse = null
)
```

and seems like we are also missing `pageLimit`?
```suggestion
        skip + pageLimit,
        pagelimit,
```